### PR TITLE
README: Add docs on manually publishing to RubyGems and bumping Homebrew

### PR DESCRIPTION
**Pull request by issyl0**
- We should automate these via GitHub Actions, but I'll leave that fun
  exercise for someone else.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,3 +43,20 @@ gds govuk connect --help
...
+```bash
+gem build .
+gem publish govuk-connect-<version>.gem # Credentials are in govuk-secrets/pass/2ndline
+git tag <version> # eg 0.0.3
+git push origin <version>
+```
+
+To raise a PR to bump the Homebrew formula version, run:
+
+`brew bump-formula-pr govuk-connect --url=https://rubygems.org/downloads/govuk-connect-<version>.gem`
```
Comment (benthorner) [inline README.md]: Does this need credentials too?

Comment (issyl0) [reply README.md]: No, although now you mention it we should pass `--no-fork` as we should all have write access to the repo and don't have to fork it to our personal accounts.

Comment (issyl0) [reply README.md]: I just gave the `GOV.UK Production` team write access to `alphagov/homebrew-gds`, so we can raise and merge our own PRs. So `--no-fork` will work.

Comment (issyl0) [reply README.md]: Fixed.

**Pull request merged**